docs/sampler.md

```diff
@@ -119,16 +119,18 @@ The reference traces are stored in `data/traces/reference` folder of this reposi
 `preprocessed.tar.gz` contains the preprocessed traces for the original Azure trace for day 1, 09:00:00-11:30:00 (150
 minutes total).
 
-`sampled.tar.gz` contains the sampled traces for preprocessed trace from `preprocessed.tar.gz`. Sample sizes are 50-3k
-functions with step 50 and 3k-24k with step 1k.
+`sampled.tar.gz` contains the sampled traces for preprocessed trace from `preprocessed.tar.gz`. Sample sizes are 10-200 functions with step 10, 200-3k with step 50, and 3k-24k with step 1k.
```

Comment:
I've noticed that we have a different name for the archive (sampled_150 instead of sampled). Can you please fix it?